SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -65,6 +65,8 @@ FArchive& FSpatialNetBitReader::operator<<(UObject*& Value)
 				improbable::GetFullPathFromUnrealObjectReference(ObjectRef, FullPath);
 				UE_LOG(LogSpatialNetBitReader, Verbose, TEXT("Object ref did not map to valid object. Streaming level not loaded or actor deleted. Will be set to nullptr: %s %s"),
 					*ObjectRef.ToString(), FullPath.IsEmpty() ? TEXT("[NO PATH]") : *FullPath);
+
+				UnresolvedRefs.Add(ObjectRef.GetLevelReference());
```

Comment:
Add a TODO and a ticket to figure out how to distinguish between streaming level not loaded and actor deleted cases.